.github/workflows/golang.yaml

```
@@ -54,6 +54,12 @@ on:
         default: 'vendor,.github'
         type: string
         description: Comma separated list of directories where vulterability check is skipped
+      install-dvc:
```

```suggestion
      pull-dvc-datasets:
```

or `use-dvc` or `use-dvc-datasets`?

---

I like more `pull-dvc-datasets`, the `use-dvc` was the first idea, but I prefer `pull-dvc-datasets`